src/Appium.Net/Appium/Service/AppiumLocalService.cs

```diff
@@ -37,7 +39,9 @@ public class AppiumLocalService : ICommandServer
         private readonly int Port;
         private readonly TimeSpan InitializationTimeout;
         private readonly IDictionary<string, string> EnvironmentForProcess;
+#if !NET48
```

Comment:
Do you have any idea when we can drop this...?
I'm not so familiar with .net version env, so it is helpful to leave some comments, or when we could drop this in README.

For example, usually .net48 could be maintained until YYYY MM DD, etc... so we will drop this then, etc..